src/components/Details/carsDetails.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { useAuthHeader } from 'react-auth-kit';
+import { useParams, useDispatch } from 'react-router-dom';
+// import { useParams, useNavigate } from 'react-router-dom';
+import { AiOutlineRight } from 'react-icons/ai';
+import Loader from '../Loader/Loader';
+import { fetchCars } from '../../redux/cars/cars';
+
+const DetailsCarScreen = () => {
+  const [loading, setLoading] = useState(true);
+  const dispatch = useDispatch();
+  const authHeader = useAuthHeader();
+  const { id } = useParams();
+  const carDetails = useSelector((state) => state.cars);
+  const cars = carDetails;
+  const carss = cars?.find((c) => c.id === parseInt(id, 10));
+
+  useEffect(() => {
+    dispatch(fetchCars(authHeader()));
+    setTimeout(() => {
+      setLoading(false);
+    }, 3800);
+  }, [dispatch]);
+
+  if (loading) {
+    return <Loader />;
+  }
+
+  if (!cars) {
+    return (
+      <>
+        <h2>Sorry</h2>
+        <p>
+          There is no car in Garage
+        </p>
+
+      </>
+    );
+  }
+
+  return (
+    <>
+      {carss && (
+        <div className="flex flex-col items-center md:justify-start justify-center w-full md:flex-row grow h-full lg:pt-20 lg:pb-10">
+          <div className="grow  md:w-5/6 flex items-center justify-center md:px-10 rounded-full aspect-square">
+            <img
+              src={carss.image?.url}
```

Comment:
The image is stored in `carss.image` not in `carss.image.url`. Please update this line

Reply:
Ok i'll do it